cdc/processor/pipeline/puller.go

```diff
@@ -41,17 +44,19 @@ type pullerNode struct {
 }
 
 func newPullerNode(
+	changefeedID model.ChangeFeedID,
```

Comment:
Can we put `changefeedID` into context Vars?

Reply:
yes, I will refine ctx.Vars in next PR